### ISSUE TITLE
Incorrect syntax for page redirects

### ISSUE THREAD

**Issue by JeffCimminoAmericanSurplus** [AmericanSurplusInc/Product-Pages#61]
When writing redirects, we're writing for URLs on the local server, which means you should trim the domain name before posting the URL. See the example below:
https://www.americansurplus.com/used-equipment/used-forklifts/used-electric-pallet-jack/
There are four parts of a URL, listed below:
1. Protocol _Defines the method our computer is using to talk to the server_ (https://)
2. Domain Name _The name of the server we're trying to access_ (www.americansurplus)
3. Top-Level Domain _The namespace the server we're trying to talk to is located in_ (.com)
4. File Path _The location on the server that we're trying to find_ (/used-equipment/used-forklifts/used-electric-pallet-jack/)
Because the redirects we're setting up are on the local server (americansurplus.com), it only cares about which file paths we're redirecting, since it can only redirect to different local files.

**TL;DR**: Change your formatting from _https://www.americansurplus.com/used-equipment/used-forklifts/used-electric-pallet-jack/_ to _/used-equipment/used-forklifts/used-electric-pallet-jack/_

**Comment by jberg418** [comment]
complete
